Content.IntegrationTests/Tests/GameRules/SecretStartsTest.cs

```diff
@@ -21,7 +21,10 @@ public async Task TestSecretStarts()
 
         await server.WaitAssertion(() =>
         {
-            gameTicker.StartGameRule("Secret");
+            // this mimics roundflow:
+            // rules added, then round starts
+            gameTicker.AddGameRule("Secret");
```

Comment:
idk why this isn't using a test-specific prototype but thats not this pr's problem